engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaUniversalAvroSchemaFunctionalTest.scala

```diff
@@ -87,7 +89,10 @@ class LiteKafkaUniversalAvroSchemaFunctionalTest
           sConfig(sampleBoolean, booleanSchema, integerSchema, sampleLong),
           invalidTypes(s"actual: '${typedLong.display}' expected: 'Integer'")
         ),
-        (sConfig(null, nullSchema, integerSchema, Input), invalidTypes("actual: 'Null' expected: 'Integer'")),
+        (
+          sConfig(null, nullSchema, integerSchema, Input),
+          emptyMandatoryParameter().findValid(invalidTypes("actual: 'Unknown' expected: 'Integer'"))
```

Comment:
Why didn't you just comment out this `invalidTypes` if this will always return the result of  `emptyMandatoryParameter`?

Reply:
What an interesting side effect :) Why not to just combineAll if you know that both are invalid and you are not interested in valid case? :)